app/src/androidTest/java/org/mozilla/tv/firefox/helpers/CustomPocketFeedStateProvider.kt

```diff
@@ -23,7 +23,9 @@ class CustomPocketFeedStateProvider {
 
     private val localeIsEnglish: () -> Boolean = { true }
 
-    private val pocketEndpoint = object : PocketEndpoint("VERSION", "www.mock.com".toUri(), localeIsEnglish) {
+    // Because of the endpoint overrides, the raw endpoint is unused so the arguments don't matter.
+    private val pocketEndpointRaw = PocketEndpointRaw("001", null)
```

Comment:
I don't understand what this comment means.  If the arguments don't matter, can they be removed?  I'm not clear on what's going on here.

Reply:
The test overrides any functionality using endpointRaw so the specifics of the arguments passed to the initializer of endpointRaw don't matter (because endpointRaw unused). The arguments can't be removed but maybe it'd be clearer to pass a mock in instead?

Reply:
Nevermind, didn't notice this was in the test directory.  No change necessary here.